e2e/pages/StageOne/PasswordPolicyPage.ts

```diff
@@ -22,4 +58,53 @@ export default class PasswordPolicyPage {
       }
     );
   }
+
+  // --- ON LOAD METHODS --- //
+
+  async assertButtonsVisible() {
+    assertComponentsVisible([this.backButton, this.continueButton]);
+  }
+
+  async assertTextVisible() {
+    assertComponentsVisible([
+      this.passwordPolicyHeading,
+      this.passwordText1,
+      this.passwordText2,
+      this.passwordText3,
+    ]);
+  }
+
+  async assertFormVisible() {
+    assertComponentsVisible([
+      this.createPasswordInput,
+      this.confirmPasswordInput,
+      this.createPasswordLabel,
+      this.confirmPasswordLabel,
+    ]);
+  }
+
+  async assertErrorsHidden() {
+    await assertErrorsHidden(this.createPasswordError);
+    await assertErrorsHidden(this.confirmPasswordError);
+  }
+
+  // --- FILLING IN FORM METHODS --- //
+  async assertFillCreatePasswordField(password1: string) {
+    await this.createPasswordInput.click();
+    await this.createPasswordInput.fill(password1);
+  }
+
+  async assertFillConfirmPasswordField(password2: string) {
+    await this.confirmPasswordInput.click();
+    await this.confirmPasswordInput.fill(password2);
+  }
+
+  // --- ERRORS LOGIC --- //
+  async assertCreatePasswordError(message: string) {
+    await assertErrorUtil(this.createPasswordError, message);
```

Comment:
Se comments in util files about making that util assertion more generic and re-usable